src/main/clojure/jarl/builtins/registry.clj

```diff
@@ -18,7 +18,10 @@
    "split" strings/builtin-split
    "startswith" strings/builtin-startswith
    "substring" strings/builtin-substring
+   "trim" strings/builtin-trim
+   "trim_left" strings/builtin-trim-left
    "trim_prefix" strings/builtin-trim-prefix
+   "trim_right" strings/builtin-trim-right
```

Comment:
When we have all string built-ins, it'd be good to also add a parser test, with a policy that simply exercise all these built-ins in one go, end-to-end.

Reply:
Definitely! Could use some help on that 🤓